packages/playground-editor/src/messaging.ts

```diff
@@ -0,0 +1,29 @@
+let targetUrl!: string;
```

Comment:
Do we need to non-null assert this? We're only calling the function which is guaranteed to return `string`.